### DISCUSSION TITLE
MeTube Alternative Directory Assistance

### DISCUSSION THREAD

**Discussion by JZambranoIT** [tteck/Proxmox#3257] [Help Needed] [answered]
Quick background on my storage configuration because its relevant as well as saying that I am still learning the ropes of proxmox/linux.

2 ZFS pools:
![image](https://github.com/tteck/Proxmox/assets/81186903/30ecf633-1e74-41e5-b213-dc5d5a27d725)

I typically like to create LXC/VM on my SSD backed pool then add additional storage needed on my HDD backed pool. I know its not a requirement but I like the idea of my VM/Container running on SSD for the OS then using HDD for media use.

Anyways I ran the MeTube script with all the default values and designated my SSD pool for storage. After some testing I wanted to add 100GB of storage to the container so I can download some larger playlist.

I have a two part question.

1. What path do I use when creating mount points? 
![image](https://github.com/tteck/Proxmox/assets/81186903/6f2f9508-401f-4d11-8a0a-0e2e36d96efe)

2. Once created how do I update the LXC to use this new mount point to store downloads?

I have tried searching but most google results are referring to updating environment variables and setting a new directory path for MeTube but I don't see that as an option when creating the LXC using the "Advanced" configuration of the script.

**Comment by joon-im** [comment, marked as answer]
To change the mount point, go to Proxmox and open the console of the MeTube LXC. Go to the `/opt/metube` directory and then enter `nano .env`. Change the desired environment variables to something like this: 

```
DOWNLOAD_DIR=/your_mount_point/metube
STATE_DIR= /your_mount_point/metube/.metube
TEMP_DIR=/your_mount_point/metube
YTDL_OPTIONS={"trim_file_name":30}
```

Save the file and restart the container.

**Reply by JZambranoIT** [reply]
Awesome this worked!
